lib/components/index.js

```diff
@@ -6,6 +6,7 @@ export { default as Checkbox } from './base/Checkbox.vue'
 export { default as Chips } from './base/Chips.vue'
 export { default as Page } from './base/Page.vue'
 export { default as Pagination } from './base/Pagination.vue'
+export { default as ProjectCard } from './base/ProjectCard.vue'
```

Comment:
You should add the other components you had here (environmentindicator and categories)